lib/blockchain.go

```diff
@@ -143,6 +147,127 @@ type BlockNode struct {
 	Status BlockStatus
 }
 
+func _difficultyBitsToHash(diffBits uint32) (_diffHash *BlockHash) {
+	diffBigint := btcdchain.CompactToBig(diffBits)
+	return BigintToHash(diffBigint)
+}
+
+func ExtractBitcoinBurnTransactionsFromBitcoinBlock(
+	bitcoinBlock *wire.MsgBlock, bitcoinBurnAddress string, params *BitCloutParams) []*wire.MsgTx {
+
+	burnTxns := []*wire.MsgTx{}
+	for _, txn := range bitcoinBlock.Transactions {
+		burnOutput, err := _computeBitcoinBurnOutput(
+			txn, bitcoinBurnAddress, params.BitcoinBtcdParams)
+		if err != nil {
+			glog.Errorf("ExtractBitcoinBurnTransactionsFromBitcoinBlock: Problem "+
+				"extracting Bitcoin transaction: %v", err)
+			continue
+		}
+
+		if burnOutput > 0 {
+			burnTxns = append(burnTxns, txn)
+		}
+	}
+
+	return burnTxns
+}
+
+func ExtractBitcoinBurnTransactionsFromBitcoinBlockWithMerkleProofs(
```

Comment:
These functions appear to be only used in block_view_test. Wanna move them there?